### PR TITLE
feature/anand_rathi

### PR DESCRIPTION
**Pull request by hemendra-sakpal-algobulls**

### CODE REVIEW

File: docs/broker/anandrathi.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+* Phone Number: The Phone Number you have used to Register/Sign-Up to the AlgoBulls website.
+
+* Password: The password you have given to Register/Sign-Up to the AlgoBulls website.
+
+[![AnandRathi](imgs/sign-in-2.png "Click to Enlarge or Ctrl+Click to open in a new Tab") ](imgs/sign-in-2.png)
+
+* Now go to **Broking Details** and click add broker.
+
+[![IIFL](imgs/brokingdetails.png "Click to Enlarge or Ctrl+Click to open in a new Tab") ](imgs/brokingdetails.png)
```
Comment (akhil-jain-algobulls) [inline docs/broker/anandrathi.md:57]: `IIFL` ?

File: docs/broker/anandrathi.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+* Choose the broker.
+
+[ ![Acagarwal](imgs/anand_rathi/choose_anand_rathi.png "Click to Enlarge or Ctrl+Click to open in a new Tab") ](imgs/anand_rathi/choose_anand_rathi.png)
```
Comment (akhil-jain-algobulls) [inline docs/broker/anandrathi.md:61]: `Acagarwal` ?

File: docs/broker/anandrathi.md (hunk 1 above)
Comment (hemendra-sakpal-algobulls) [reply docs/broker/anandrathi.md:57]: Done

File: docs/broker/anandrathi.md (hunk 2 above)
Comment (hemendra-sakpal-algobulls) [reply docs/broker/anandrathi.md:61]: Done

**Review by akhil-jain-algobulls — APPROVED** [review]

**Pull request merged**